acitoolkit/acisession.py

```diff
@@ -30,10 +30,14 @@
 """  This module contains the Session class that controls communication
      with the APIC.
 """
+import base64
 import copy
 import json
 import logging
+import requests
+import socket
 import ssl
+import sys
 import threading
 import time
 import socket
```

Comment:
We should also remove the duplicate imports here for base64, requests, socket and sys.

Reply:
Yes, good point - will fix.